Assets/Scripts/CharacterControl/Character.cs

```diff
@@ -36,12 +39,14 @@ private void ToggleMode()
                 movement.SetConfig(catMovement);
                 interactableHandler.DropPickup();
                 interactableHandler.enabled = false;
+                this.GetComponent<SpriteRenderer>().sprite=human;
```

Comment:
This should be the cat sprite, not human sprite. Additionally, you never change the size of the collider, despite making a field for it.

Reply:
Also, remove the `this` keyword since its not needed in this case. It's only needed when a local variable overlaps with one of the class's members (and is thus syntactically required). Take a look at [this SO answer](https://stackoverflow.com/a/2411283), specifically case 1. It's about Java, but the same logic applies in C#.

Reply:
Also, add spaces before and after your assignment operator. This rule applies to all other operators, e.g. `+`, `-`, `/`, etc.